docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,498 @@
+
+## Alerting
+
+_All Alerting features are generally available in all editions of Grafana_
+
+We’ve made a number of improvements to simplify the alert rule creation process as well as improvements to contact points and alert management. For all the details, refer to our [Alerting documentation]({{< relref "../alerting" >}}).
+
+### Alert rules
+
+We’ve made the following changes to alert rules.
+
+#### Alert instance routing preview
+
+_This feature is for Grafana-managed alert rules only._
+
+Preview how your alert instances will be routed if they fire while you're creating your alert rule. View routing for each Alertmanager you have configured to receive Grafana-managed alerts and if required, you can easily make adjustments to your custom labels to change the way your alert instances are routed.
+
+{{< figure src="/media/docs/alerting/alert-routing-preview.png" max-width="750px" caption="Alert instance routing preview" >}}
+
+#### Alert rule types
+
+Enables you to switch to a data source-managed alert rule if your data source is configured to support alert rule creation (Ruler API enabled). By default, the alert rule type is Grafana-managed.
+
+{{< figure src="/media/docs/alerting/alert-rule-types.png" max-width="750px" caption="Alert rule types" >}}
+
+#### UI improvements
+
+- **Alert evaluation behavior**: New UI components for creating a folder and adding an evaluation group along with improved text and validation.
+- **Alert Rule list page**: the process of creating recording rules (More drop down) is now separate from Grafana-managed and data source-managed alert rules (**+New alert rule**)
+  .
+- **Annotations display**: Adding a summary, description, and runbook URL as annotations are now optional. The dashboard and panel names are now also linked directly, making it easier to access.
+- **View YAML button**: Displays alert rule configuration in YAML format on the Grafana-managed alert rules form as well as Grafana-managed provisioned and non-provisioned Alert Rule detail view.
+- **Queries and expressions**: Several improvements have been made to the display of queries and expressions, including making Add expression a dropdown and moving Conditions to the header.
+- **Min interval option**: Improves control over query costs and performance by enabling you to adjust the minimum resolution of the data used in your alerting queries.
+- **In-app guidance for alert rule creation**: Learn about how to create your alert rules interactively with in-app guidance for additional context and links out to our Alerting documentation.
+- **Support for toggling common labels**: Toggle between showing or hiding labels for each individual alert instance.
+
+### Contact points
+
+We’ve made the following changes to contact points.
+
+#### Additional contact points for external Alertmanager
+
+Adds support for the following contact points when using an external Alertmanager:
```

Comment:
```suggestion
We've added support for the following contact points when using an external Alertmanager:
```

Reply:
again, fine for me. interesting though -- i'm used to conventions that always use verbs to avoid extra words like you can, we have etc. because this is stating the obvious...is our convention to use you can, we have, etc.?

Reply:
If this were a stand-alone section, I'd tend to agree, but overall the style of this document tends to lean a teeny bit more marketing in style, so it's less concise.